src/problem-3.js

```diff
@@ -0,0 +1,31 @@
+const houses = [
+    { name: "Targaryen", motto: "Fire and Blood" },
+    { name: "Stark", motto: "Winter is Coming" },
+    { name: "Bolton", motto: "Our Blades Are Sharp" },
+    { name: "Greyjoy", motto: "We Do Not Sow" },
+    { name: "Tully", motto: "Family, Duty, Honor" },
+    { name: "Arryn", motto: "As High as Honor" },
+    { name: "Lannister", motto: "Hear Me Roar!" },
+    { name: "Tyrell", motto: "Growing Strong" },
+    { name: "Baratheon", motto: "Ours is the Fury" },
+    { name: "Martell", motto: "Unbowed, Unbent, Unbroken" }
+];
+
+function GOT(house) {
+    let answer;
+    for (let i = 0; i < houses.length; i++) {
+        if (houses[i].name === house) {
+            answer = houses[i].motto
+        }
+    }
+    return answer;
+};
+
+const GOTES6 = (house) => {
+    return houses.find(x => x.name === house).motto
```

Comment:
This will produce an error if the house was not found. Try adding a test case for that first to see :)